### PR TITLE
Feature on 98

### PR DESCRIPTION
**Pull request by trololov**

### CODE REVIEW

**Comment by trololov** [comment]
@niekert can you please review for me?

File: lib/messagebird.js [hunk 1]
```diff
@@ -93,6 +93,7 @@ module.exports = function (accessKey, timeout, features) {
     options.headers = extend(options.headers || {}, requestParams.headers || {})
 
     request = http.request(options);
+    // console.log(options)
```
Comment (niekert) [inline lib/messagebird.js]: leftover console.log

File: lib/messagebird.js [hunk 2]
```diff
@@ -875,6 +876,111 @@ module.exports = function (accessKey, timeout, features) {
 
     },
 
+    callflows: {
+
+      /**
+       * Lists existing call flows.
+       *
+       * @param {Function} callback
```
Comment (niekert) [inline lib/messagebird.js]: This is missing the `page` and `perpage` params from the JSDoc

File: lib/messagebird.js [hunk 3]
```diff
@@ -875,6 +876,111 @@ module.exports = function (accessKey, timeout, features) {
...
+          method: 'GET',
+          path: '/call-flows',
+          params: params},
+          callback);
+      },
+
+      /**
+       * Creates a new call flow, params are mandatory.
+       *
+       * @param {String} name
```
Comment (niekert) [inline lib/messagebird.js]: Seems like this function no longer has a name parameter

File: lib/test.js [hunk 4]
```diff
@@ -26,7 +26,7 @@ var cache = {
     recipients: [number],
     type: 'sms',
     body: 'Test message from node ' + process.version,
-    gateway: 2
+    // gateway: 2
```
Comment (niekert) [inline lib/test.js]: Maybe forgot to uncomment this?

**Pull request merged**